build/dependencies.yaml

```diff
@@ -215,7 +215,7 @@ dependencies:
 
   # metadata-concealment: bump this one first
   - name: "metadata-concealment"
-    version: "1.6"
+    version: "1.7"
```

Comment:
unrelated change?

Reply:
Right, thanks for pointing out this. Just fixed.